### PR TITLE
sunlight: update issuers handling

### PR DESCRIPTION
**Pull request by FiloSottile**

### CODE REVIEW

**Review by mhutchinson — APPROVED** [review]
LGTM for alignment with the tiles spec 👍

File: sunlight.md [hunk 1]
```diff
@@ -269,40 +266,43 @@ tile. Data tiles are sequences of the following structure, one per entry.
...
-The issuers bundle is served as at
+The issuers are served at
 
-    <monitoring prefix>/issuers.pem
+    <monitoring prefix>/issuer/<fingerprint>
 
-with `Content-Type: text/plain; charset=utf-8`.
+with `Content-Type: application/pkix-cert`.
+
+`<fingerprint>` is the lowercase hex-encoded SHA-256 hash of the ASN.1 encoding
```
[the remark this answers is not in the document]
Comment (FiloSottile) [reply sunlight.md:296]: This was intentional. If a chain is submitted with an issuer that's invalid DER, I think we should hash it as is, not re-encode it as DER.

File: sunlight.md [hunk 2]
```diff
@@ -269,40 +266,43 @@ tile. Data tiles are sequences of the following structure, one per entry.
 ```
 struct {
 	TimestampedEntry timestamped_entry;
-	select(entry_type) {
+	Fingerprint certificate_chain<0..2^16-1>;
```
[the remark this answers is not in the document]
Comment (FiloSottile) [reply sunlight.md:269]: Hmm, this way it was ending up right after issuer_key_hash, but I suppose the pre_certificate is the first certificate in the chain. I'll move it.

File: sunlight.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (mcpherrinm) [reply sunlight.md:296]: I think that's valid in a BER vs DER context, where DER is a subset of BER, but there's other ASN.1 encoding rules like JSON and XML.  In the x509 context, I don't think anyone is ever going to be confused by this, so this is just spec nit-picking

[the remark this answers is not in the document]
Comment (FiloSottile) [reply sunlight.md:296]: Clarified that the hashed encoding has to match exactly what the client submitted (which clarifies that it doesn't _have_ to be DER but also can't be some other ASN.1 encoding).

**Pull request merged**